client/blocks/login/social.scss

```diff
@@ -16,13 +16,6 @@
 .auth-form__social.is-login {
```

Comment:
All the deleted sections in this block have been removed because they are direct copies of styles below under a broader scope [`.auth-form__social`](https://github.com/Automattic/wp-calypso/pull/100807/files#diff-934caa216b18163d15b19bde2bf9c56db6984aba9bed8cd927226eb56cedee12L243)